frontend/packages/erd-core/src/components/ERDRenderer/ERDRenderer.tsx

```diff
@@ -82,15 +84,19 @@ export const ERDRenderer: FC<Props> = ({
                         nodes={nodes}
                         edges={edges}
                       />
-                      <div className={styles.toolbarWrapper}>
-                        <Toolbar />
-                      </div>
                       <TableDetailDrawer />
                     </>
                   )}
                 </TableDetailDrawerRoot>
               </main>
             </div>
+
+            {errorObjects.length > 0 || (
```

Comment:
Because of a double negative, following may be easier to understand: 

```suggestion
            {errorObjects.length === 0 && (
```